plugins/csl-tools/CMakeLists.txt

```
@@ -0,0 +1,57 @@
+# ------------------------------------------------------------------------------------------------ #
+#                                This file is part of CosmoScout VR                                #
+# ------------------------------------------------------------------------------------------------ #
+
+# SPDX-FileCopyrightText: German Aerospace Center (DLR) <cosmoscout@dlr.de>
+# SPDX-License-Identifier: MIT
+
+option(CSL_TOOLS "Enable compilation of this plugin" ON)
+
+if (NOT CSL_TOOLS)
+  return()
+endif()
+
+# build plugin -------------------------------------------------------------------------------------
+
+file(GLOB SOURCE_FILES src/*.cpp)
+
+# Resource files and header files are only added in order to make them available in your IDE.
+file(GLOB HEADER_FILES src/*.hpp)
+
+add_library(csl-tools SHARED
+  ${SOURCE_FILES}
+  ${HEADER_FILES}
+  ${RESOURCE_FILES}
+)
```

There are no resource files here.
```suggestion
  ${HEADER_FILES}
)
```